shipctl/x86_64/Ubuntu_16.04/shipctl

```diff
@@ -28,6 +28,8 @@ elif [ "$shipctl_command" == "service" ]; then
   execute_shipctl_command shippable_service $arguments
 elif [ "$shipctl_command" == "replace" ]; then
   execute_shipctl_command shippable_replace $arguments
+elif [ "$shipctl_command" == "bump_version" ]; then
```

Comment:
This will not be required anymore once we move the `bump_version` method to `utility.sh`.